site/en/blog/navigateevent-intercept/index.md

```diff
@@ -78,7 +78,7 @@ In single page apps, you can now control whether the browser handles scrolling t
 
 By default, the browser will attempt to handle scrolling automatically, once the intercept hander has fulfilled. If you want to handle the scrolling yourself, set `scroll` to `"manual"`, then call `NavigateEvent.scroll()` when the browser should attempt to set the scroll position.
 
-```js/4,6
+```js/2,8
```

Comment:
doh, thanks for catching this. I saw that I needed to update this, but I guess I just forgot.

Reply:
np